### ISSUE TITLE
Can't save blobs for iOS 13.0 / iPhone XS (11,2)

### ISSUE THREAD

**Issue by Axel29** [airsquared/blobsaver#108]
**Describe the bug**
Can't save Blobs for iOS 13 on iPhone XS (11.2).
I've masked some information in the debug logs like the path to my blobsaver or my ECID.

**Screenshots**
![image](https://user-images.githubusercontent.com/3015249/65379302-47ef2100-dcc6-11e9-80ba-9e434abb056c.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - App version: 2.3.2

**Log(s)**
```
Running: [xxxxxx\.blobsaver_bin\tsschecker, --generator, 0x1111111111111111, --nocache, -d, iPhone11,2, -s, -e, XXXXXXXXXXXX, --save-path, xxxxxxxxx\shshs, --boardconfig, D321AP, -i, 13.0]
Version: 8ce7eaf0a731c39eb1e08ed6725303d6d6dd2eee - 340
[TSSC] manually specified generator "0x1111111111111111"
[TSSC] manually specified ECID to use, parsed "XXXXXXXXXXXX" to dec:3752157006397486 hex:XXXXXXXXXXXX
[TSSC] opening firmware.json
[DOWN] downloading file https://api.ipsw.me/v2.1/firmwares.json/condensed
[JSON] counting elements
[JSON] parsing elements
[TSSC] got firmware URL for iOS 13.0 build 17A577
[TSSC] opening Buildmanifest for iPhone11,2_13.0_17A577
[LFZP] downloading BuildManifest.plist from http://updates-http.cdn-apple.com/2019FallFCS/fullrestores/061-08680/B9063AE6-C875-11E9-AC7C-A0936C418163/iPhone11,2,iPhone11,4,iPhone11,6_13.0_17A577_Restore.ipsw

[Error] [LFZP] failed to open URL
[Error] [TSSC] ERROR: could not get BuildManifest for firmware URL http://updates-http.cdn-apple.com/2019FallFCS/fullrestores/061-08680/B9063AE6-C875-11E9-AC7C-A0936C418163/iPhone11,2,iPhone11,4,iPhone11,6_13.0_17A577_Restore.ipsw

[Error] [TSSC] checking TSS status failed!
```

**Comment by Axel29** [comment]
Seems to be the same as #107.

**Comment by airsquared** [comment]
This issue has been fixed in v2.4.0.

**Comment by Axel29** [comment]
Perfect, thank you!
